src/Deposit.js

```diff
@@ -1369,9 +1387,9 @@ export default class Deposit {
   /**
    * Notify the contract that the signers have failed to produce a redemption proof.
    */
-  async notifyRedemptionProofTimeout() {
+  async notifyRedemptionProofTimedOut() {
     await EthereumHelpers.sendSafely(
-      this.contract.methods.notifyRedemptionProofTimdOout()
+      this.contract.methods.notifyRedemptionProofTimedOout()
```

Comment:
Extra 'o' in Out?  Didn't pull code, only looking on Github.com so can't search, sorry if this is correct!

Reply:
Whew. Yep.

Reply:
Fixed!